v4/source/introduction.yaml

```diff
@@ -214,23 +214,193 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          one: {
+            id: t('combined_system_message.added_to_team.one'),
+            defaultMessage: '{firstUser} **added to the team** by {actor}.',
+          },
+          one_you: {
+            id: t('combined_system_message.added_to_team.one_you'),
+            defaultMessage: 'You were **added to the team** by {actor}.',
+          },
+          two: {
+            id: t('combined_system_message.added_to_team.two'),
+            defaultMessage: '{firstUser} and {secondUser} **added to the team** by {actor}.',
+          },
+          many_expanded: {
+            id: t('combined_system_message.added_to_team.many_expanded'),
+            defaultMessage: '{users} and {lastUser} were **added to the team** by {actor}.',
+          }
+        ```
       - authentication_challenge
+        ```
+        {
+          token: <token-id>
+        }
+        ```
       - channel_converted
+        ```
+        
+        ```
       - channel_created
+        ```
+        {
+          'broadcast': {
+            'omit_users': None,
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>',
+            'user_id': '<user_id>'
+          },
+          'data': {
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'
+          },
+          'event': 'channel_created',
+          'seq': 9
+        }
+        ```
       - channel_deleted
+        ```
+        {
+          'event': 'WebsocketEvents.CHANNEL_DELETED',
```

Comment:
`WebsocketEvents.CHANNEL_DELETED` is a constant that stands in for a string. We'd want to put the string here